lib/commands/init.js

```diff
@@ -72,6 +72,10 @@ module.exports = Command.extend({
       return Promise.reject(new SilentError('We currently do not support a name of `' + packageName + '`.'));
     }
 
+    if (blueprintOpts.blueprint[0] === '.') {
+      blueprintOpts.blueprint = path.resolve(process.cwd(), blueprintOpts.blueprint);
```

Comment:
at first, I thought this was a mistake. But I believe it to be correct, as you want:

```js
cd app/ && ember --blueprint=../blueprints/foo
```

to be relative the cwd, not the project root.

:+1: